src/components/registrering/foreslatt-profilering.tsx

```diff
@@ -20,8 +20,13 @@ export function ForeslattProfilering(props: Props) {
 
     return (
         <Panel border className="profilering_boks">
-            <Ingress>Forslag om brukers muligheter og behov (resultat fra profilering)</Ingress>
-            <BodyShort>{innsatsgruppeBeskrivelse(ordinaerRegistrering.profilering.innsatsgruppe)}</BodyShort>
+            <Heading level="4" size="small">
```

Comment:
Den var bare veldig off fra resten av boksen, så jeg endret litt - men har spurt om dette i tråden så dette er vel egentlig ikke så viktig. Fant ikke noe sånn "standard" i Aksel for jeg vet ikke helt hva denne boksen klassifiserer som.